Makefile

```diff
@@ -423,10 +423,12 @@ utils/config.ml: utils/config.mlp config/Makefile
 	    -e 's|%%ARCH%%|$(ARCH)|' \
 	    -e 's|%%MODEL%%|$(MODEL)|' \
 	    -e 's|%%SYSTEM%%|$(SYSTEM)|' \
+	    -e 's|%%EXT_EXE%%||' \
```

Comment:
I'm fairly sure this logic isn't correct - Cygwin should have .exe and possibly also Linux cross-compiling using mingw? (both of which use configure, not Makefile.nt)